scripts/generate_hyperparameters.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+numreplicates=10
```

Comment:
Let's not hard code any arguments in this script.  The script should operate only on input files or command line arguments.